src/routes/(app)/Drawer.svelte

```diff
@@ -12,53 +14,85 @@
 
 <input id="main-drawer" type="checkbox" class="drawer-toggle" bind:checked />
 <div class="drawer-side z-20">
-  <label for="main-drawer" aria-label="close sidebar" class="drawer-overlay"
-  ></label>
-  <ul class="menu min-h-full min-w-60 bg-base-200 pr-6 font-semibold">
-    {#each routes as route (route.title)}
-      {#if !route.accessRequired || isAuthorized(route.accessRequired, $page.data.user)}
-        {#if route?.children?.length}
-          <li class="py-1">
-            <details open>
-              <summary class="active:!bg-primary/10">
+  <label for="main-drawer" aria-label="close sidebar" class="drawer-overlay" />
+
+  <ul class="menu min-h-full min-w-60 bg-base-200 p-0 font-semibold">
+    <div class="navbar justify-between p-2 pl-6 pr-4 shadow-md">
+      <div class="flex gap-2">
+        <DsekLogo className="size-6 text-primary" />
+        <p class="font-thin">D-SEKTIONEN</p>
+      </div>
+      <div class="flex *:px-1">
+        <label class="btn btn-ghost swap swap-rotate btn-sm rounded *:text-xl">
+          <!-- this hidden checkbox controls the state -->
+          <input type="checkbox" data-toggle-theme="dark,light" />
+
+          <!-- moon icon -->
+          <span class="swap-on i-mdi-weather-night" />
+
+          <!-- sun icon -->
+          <span class="swap-off i-mdi-weather-sunny" />
+        </label>
+        <div class="btn btn-ghost btn-sm rounded">
+          {#if languageTag() === "sv"}
+            <a href={i18n.route($page.url.pathname)} hreflang="en">EN</a>
+          {:else if languageTag() === "en"}
+            <a href={i18n.route($page.url.pathname)} hreflang="sv">SV</a>
+          {/if}
+        </div>
```

Comment:
These correspond to the `DarkLightToggle` and `LanguageSwitcher` components. I should probably use those instead, but they don't really work (I had to make some small design changes, i.e making them smaller).